config/vault/openshift/route.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: vault
+  namespace: system
+spec:
+  to:
+    kind: Service
+    name: spi-vault
+    weight: 100
+  port:
+    targetPort: 8200
```

Comment:
If we're changing the port to 8201, we should do that here, too, right?

Reply:
yes